From c71d81045b663db430b4e06e0cdeda9e001e120a Mon Sep 17 00:00:00 2001
From: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Date: Sun, 16 Dec 2018 23:05:12 +0200
Subject: [PATCH 006/235] arm64: dts: freescale: lx2160a: Inphi in112525_s03
 mdio node

Add Inphi retimer phyid in the mdio node, solving the probe issue
for this non-standard clause-45 device.

Signed-off-by: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
index c00839541dfd..59e32a9a75e0 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -174,11 +174,9 @@
 };
 
 &emdio2 {
-	inphi_phy: emdio2_phy@0 {
-		compatible = "ethernet-phy-ieee802.3-c45";
+	inphi_phy: ethernet-phy@0 {
+		compatible = "ethernet-phy-id0210.7440";
 		reg = <0x0>;
-		interrupts = <0 9 IRQ_TYPE_EDGE_FALLING>,
-			<0 10 IRQ_TYPE_EDGE_FALLING>;
 	};
 };
 
-- 
2.17.1

